Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -127,57 +136,51 @@ public bool IsDirty
             set => isDirty = value;
         }
 
+        /// <summary>
+        /// Public access for the renderer(s) that should be affected by the primitive.
+        /// </summary>
+        public List<Renderer> Renderers => renderersCache;
+
         /// <summary>
         /// Adds a renderer to the list of objects this clipping primitive clips.
         /// </summary>
-        /// <param name="_renderer">The renderer to add.</param>
-        public void AddRenderer(Renderer _renderer)
+        /// <param name="renderer">The renderer to add.</param>
+        public void AddRenderer(Renderer renderer)
```

Comment:
Is the member name "renderer" allowed now? It use to cause a collision with Unity's Component type.

Reply:
I think so as there is no complain from VS or during build time and also there are other places where `renderer` is used as a variable name.